### PR TITLE
[2.11.x] DDF-3478  Removes outdated Simple Search UI documentation

### PR DESCRIPTION
**Pull request by vinamartin**
#### What does this PR do?
Removes outdated Simple Search UI documentation. 
The configuration for Simple Search UI is improved by this PR: https://github.com/codice/ddf/pull/2724 however, the functionality still works in 2.11.x.

To test this: open simple search and verify it works out of the box.
Then using the docs, navigate to its configuration and configure it. Verify everything works as expected. 

#### Who is reviewing it? 
@ahoffer @mcalcote @emanns95 @Lambeaux 

#### Choose 2 committers to review/merge the PR. 
@rzwiefel 
@ricklarsen - Documentation

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: distribution/docs/src/main/resources/content/_transformers/searchui-xformer.adoc [hunk 1]
```diff
@@ -1,79 +0,0 @@
...
-:summary: Provides results in html format and a convenient, simple querying user interface.
-
-The Simple SearchUI is a `QueryResponseTransformer` that provides results in html format and a convenient, simple querying user interface.
-It is primarily used as a test tool and verification of configuration.
-The left pane of the Simple SearchUI contains basic fields to query the Catalog and other Sources.
-The right pane consists of the results returned from the query.
-
-===== Installing Simple SearchUI
-
-This transformer is installed by default with a standard installation in the ${ddf-catalog} application as the feature, `catalog-transformer-ui`.
```
Comment (emanns95) [inline distribution/docs/src/main/resources/content/_transformers/searchui-xformer.adoc]: Do we no longer include this type of information?

Comment (vinamartin) [reply distribution/docs/src/main/resources/content/_transformers/searchui-xformer.adoc]: It's no longer a transformer, so it felt right to get rid of it. 
Also the feature it describes here literally does not exist.

Comment (emanns95) [reply distribution/docs/src/main/resources/content/_transformers/searchui-xformer.adoc]: Oh didn't realize this was for a transformer not the simple search itself. Cool 👍

**Comment by vinamartin** [comment]
build now

**Pull request merged**